### PR TITLE
Updated items to current patch

### PR DESCRIPTION
**Pull request by imaGecko**
Applied item changes from patch 7.06 to 7.03
Kept old progression wherever possible.
Hand of Midas still has the same max level values
Changed Heart Progression to prevent it from completely breaking the game

### CODE REVIEW

**Comment by Haganeko** [comment]
Did you redo the progression based on our formulas and current balance adjustments, or just keep the upgraded values as is?

**Comment by imaGecko** [comment]
I kept the same progression. An item with 20/40/60/80/100 would become 15/35/55/75/95
But I had to make some exceptions that I listed.

File: game/resource/English/ability/items/tooltip_hand_of_midas.txt [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+"DOTA_Tooltip_Ability_item_hand_of_midas_Note0"                               "The gold given is reliable gold (you do not get the normal creep bounty)."
+"DOTA_Tooltip_Ability_item_hand_of_midas_Note1"                               "Experience gained by using Transmute is not shared."
+"DOTA_Tooltip_Ability_item_hand_of_midas_bonus_attack_speed"                  "+$attack"
+"DOTA_Tooltip_Ability_item_hand_of_midas_transmute_cast_range_tooltip"        "TRANSMUTE CAST RANGE:"
+
 "DOTA_Tooltip_Ability_item_hand_of_midas_2"             "Hand of Midas"
 "DOTA_Tooltip_Ability_item_recipe_hand_of_midas_2"      "Hand of Midas Recipe"
-"DOTA_Tooltip_Ability_item_hand_of_midas_2_Description"                             "Active: Transmute - Kills a non-hero target for %bonus_gold% gold and %xp_multiplier%x experience.  Cannot be used on Ancient Creeps."
+"DOTA_Tooltip_Ability_item_hand_of_midas_2_Description"                             "Active: Transmute - Kills a non-hero target for %bonus_gold% gold and 1.75/3/4.5x experience.  Cannot be used on Ancient Creeps."
```
Comment (kallleeeeh) [inline game/resource/English/ability/items/tooltip_hand_of_midas.txt]: Keep %xp_multiplier% here. No hard coded numbers in tooltips

Comment (imaGecko) [inline game/resource/English/ability/items/tooltip_hand_of_midas.txt]: It doesn't display the numbers correctly. 1.75 will be shown as 1.8, so this way is exactly how they solved it in the original dota 2 tooltips

Comment (imaGecko) [inline game/resource/English/ability/items/tooltip_hand_of_midas.txt]: Double-checked, doesn't change anything. Pressing alt changes damage numbers of abilities to include spell amp, it doesn't make numbers more or less exact.

**Pull request merged**